### PR TITLE
fix: LEAP-872, LEAP-935, LEAP-942, including moving the

### PR DESCRIPTION
**Pull request by deppp**
auto-accept checkbox to the bottom bar, fixing reject all button not working, and removing the regions used for interactivity

### PR fulfills these requirements
- [x] Commit message(s) and PR title follows the format `[fix|feat|ci|chore|doc]: TICKET-ID: Short description of change made` ex. `fix: DEV-XXXX: Removed inconsistent code usage causing intermittent errors`
- [ ] Tests for the changes have been added/updated (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [ ] Best efforts were made to ensure docs/code are concise and coherent (checked for spelling/grammatical errors, commented out code, debug logs etc.)
- [x] Self-reviewed and ran all changes on a local instance (for bug fixes/features)

#### Change has impacts in these area(s)
_(check all that apply)_
- [x] Product design
- [ ] Backend (Database)
- [ ] Backend (API)
- [x] Frontend

### Describe the reason for change
_(link to issue, supportive screenshots etc.)_

#### What does this fix?
_(if this is a bug fix)_

#### What is the new behavior?
_(if this is a breaking or feature change)_

#### What is the current behavior?
_(if this is a breaking or feature change)_

#### What libraries were added/updated?
_(list all with version changes)_

#### Does this change affect performance?
_(if so describe the impacts positive or negative)_

#### Does this change affect security?
_(if so describe the impacts positive or negative)_

#### What alternative approaches were there?
_(briefly list any if applicable)_

#### What feature flags were used to cover this change?
_(briefly list any if applicable)_

### Does this PR introduce a breaking change?
_(check only one)_
- [ ] Yes, and covered entirely by feature flag(s)
- [ ] Yes, and covered partially by feature flag(s)
- [ ] No
- [ ] Not sure (briefly explain the situation below)

### What level of testing was included in the change?
_(check all that apply)_
- [ ] e2e
- [ ] integration
- [ ] unit

### Which logical domain(s) does this change affect?
_(for bug fixes/features, be as precise as possible. ex. Authentication, Annotation History, Review Stream etc.)_

### CODE REVIEW

File: web/libs/editor/src/components/ImageView/SuggestionControls.js [hunk 1]
```diff
@@ -84,14 +84,22 @@ export const SuggestionControls = observer(({ item, useLayer }) => {
           cornerRadius={16}
         />
         <ControlButton
-          onClick={() => item.annotation.rejectSuggestion(item.id)}
+          onClick={() => {
+            const ann = item.annotation;
+            ann.rejectSuggestion(item.id);
+            ann.deleteAllDynamicregions(true);
```
Comment (deppp) [inline web/libs/editor/src/components/ImageView/SuggestionControls.js:90]: Fix for LEAP-942

File: web/libs/editor/src/components/ImageView/SuggestionControls.js [hunk 2]
```diff
@@ -84,14 +84,22 @@ export const SuggestionControls = observer(({ item, useLayer }) => {
...
           iconColor="#fff"
           icon={IconCross}
         />
         <ControlButton
           x={32}
-          onClick={() => item.annotation.acceptSuggestion(item.id)}
+          onClick={() => {
+            const ann = item.annotation;            
+            ann.acceptSuggestion(item.id);
+            ann.deleteAllDynamicregions(true);
```
Comment (deppp) [inline web/libs/editor/src/components/ImageView/SuggestionControls.js:101]: Fix for LEAP-942

File: web/libs/editor/src/stores/Annotation/Annotation.js [hunk 3]
```diff
@@ -1338,14 +1326,14 @@ export const Annotation = types
       Array.from(self.suggestions.keys()).forEach((id) => {
         self.acceptSuggestion(id);
       });
-      self.deleteAllDynamicregions(isFF(FF_DEV_1284));
+      self.deleteAllDynamicregions(true);
     },
 
     rejectAllSuggestions() {
-      Array.from(self.suggestions.keys).forEach((id) => {
+      Array.from(self.suggestions.keys()).forEach((id) => {
```
Comment (deppp) [inline web/libs/editor/src/stores/Annotation/Annotation.js:1333]: Fix for LEAP-935

**Comment by hlomzik** [comment]
Closed in favor of https://github.com/HumanSignal/label-studio/pull/5714

**Pull request closed without merging**